src/link/MachO.zig

```diff
@@ -462,6 +465,11 @@ pub fn flushModule(self: *MachO, arena: Allocator, tid: Zcu.PerThread.Id, prog_n
         };
     }
 
+    if (comp.fuzzer_lib) |fuzzer_lib| {
+        _ = fuzzer_lib.full_object_path;
+        log.err("TODO macho linking code for adding libfuzzer", .{});
+    }
+
```

Comment:
I'm confused, what do you expect  the linker to do here? I thought it should simply put the path to the fuzzing lib on the linker line? Is there more to it? Also, how come this is only in the MachO linker and not ELF?

Reply:
I didn't understand until now that adding to positionals made it be parsed correctly. Thanks!